packages/fuel-indexer-schema/src/lib.rs

```diff
@@ -130,7 +130,9 @@ impl FtColumn {
                 );
                 FtColumn::Timestamp(Some(int8))
             }
-            ColumnType::Blob => FtColumn::Blob(Some(bytes[..size].to_vec())),
+            ColumnType::Blob => {
+                FtColumn::Blob(Some(fuel_indexer_types::Blob(bytes[..size].to_vec())))
```

Comment:
Should not have to wrap a `Blob` here. Using proper `impl` we can just `.into()` this